src/bootstrap_handler.rs

```diff
@@ -184,6 +194,33 @@ mod test {
         path.parent().unwrap().join(name)
     }
 
+    fn create_contacts() -> Contacts {
+        let addr = net::SocketAddrV4::new(net::Ipv4Addr::new(1,2,3,4), 8080);
+        let contact  = Contact { endpoint: Endpoint::Tcp(SocketAddr::V4(addr)) };
+        let mut contacts = Contacts::new();
+        contacts.push(contact.clone());
+        contacts
+    }
+
+    #[test]
+    fn bootstrap_read() {
+        match BootstrapHandler::new().as_mut() {
+            Ok(handler) => println!("{:?}", handler.read_bootstrap_file()),
+            Err(_) => assert!(false)
+        }
+    }
+
+    #[test]
+    fn bootstrap_write() {
```

Comment:
Although, I think at least in this test (maybe more) we should close the file and re-read, checking that the contents persist.